### PR TITLE
DOMA-3303 ticket on section 

### PR DESCRIPTION
**Pull request by nomerdvadcatpyat**
https://user-images.githubusercontent.com/52532264/178675103-ce471114-2579-4f12-8958-fb59c0b59b69.mov

### CODE REVIEW

File: apps/condo/domains/property/components/UnitInfo.tsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
 import get from 'lodash/get'
 
 import { TicketFormItem } from '@condo/domains/ticket/components/BaseTicketForm'
 import { UnitNameInput, UnitNameInputOption } from '@condo/domains/user/components/UnitNameInput'
 
-import { BuildingSection, BuildingUnitSubType } from '@app/condo/schema'
+import { BuildingFloor, BuildingSection, BuildingUnit, BuildingUnitSubType } from '@app/condo/schema'
 import { IPropertyUIState } from '@condo/domains/property/utils/clientSchema/Property'
 import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+import { FloorNameInput } from '../../user/components/FloorNameInput'
```
Comment (akarjakin) [inline apps/condo/domains/property/components/UnitInfo.tsx:14]: @condo/domains/user/components/...

File: apps/condo/domains/property/components/UnitInfo.tsx [hunk 2]
```diff
@@ -43,56 +46,122 @@ const getSectionAndFloorByUnit: IGetSectionAndFloorByUnit = (unitName, sections,
...
     const { isSmall } = useLayoutContext()
 
+    const [selectedSectionName, setSelectedSectionName] = useState<string>(get(initialValues, 'sectionName'))
+    const [selectedFloorName, setSelectedFloorName] = useState<string>(get(initialValues, 'floorName'))
+
+    const sections = useMemo(() => get(property, ['map', 'sections'], []), [property])
+    const parking = useMemo(() => get(property, ['map', 'parking'], []), [property])
+    const selectedSections = useMemo(() => {
+        if (selectedSectionName) {
+            if (selectedSectionType === SECTION_SECTION_TYPE) {
```
Comment (akarjakin) [inline apps/condo/domains/property/components/UnitInfo.tsx:102]: return selectedSectionType === SECTION_SECTION_TYPE) ? sections : parking

File: apps/condo/domains/user/components/FloorNameInput.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import Select from '@condo/domains/common/components/antd/Select'
+
+export const FloorNameInput = ({ floors, ...restSelectProps }) => {
+    const options = [...floors].reverse().map(
```
Comment (akarjakin) [inline apps/condo/domains/user/components/FloorNameInput.tsx:6]: please memoize `[...floors].reverse()`

File: apps/condo/domains/property/components/UnitInfo.tsx [hunk 4]
```diff
@@ -43,56 +46,122 @@ const getSectionAndFloorByUnit: IGetSectionAndFloorByUnit = (unitName, sections,
...
-        setSelectedUnitType?: React.Dispatch<React.SetStateAction<BuildingUnitSubType>>,
-    }): React.ReactElement
+    property: IPropertyUIState
+    form: FormInstance
+    loading: boolean
+    setSelectedUnitName: React.Dispatch<React.SetStateAction<string>>
+    setSelectedUnitType?: React.Dispatch<React.SetStateAction<BuildingUnitSubType>>
+    selectedUnitName?: string
+
+    mode?: 'unit' | 'all'
```
Comment (sitozzz) [inline apps/condo/domains/property/components/UnitInfo.tsx:67]: May be move this to `enum`?

File: apps/condo/domains/user/components/UnitNameInput.tsx [hunk 5]
```diff
@@ -10,6 +11,7 @@ import { BuildingSection, BuildingUnit, BuildingUnitSubType } from '@app/condo/s
 import { IPropertyUIState } from '@condo/domains/property/utils/clientSchema/Property'
 import { LabeledValue } from 'antd/lib/select'
 import { TrackingEventPropertiesType } from '@condo/domains/common/components/TrackingContext'
+import { getFloorsBySection } from '../../property/components/UnitInfo'
```
Comment (sitozzz) [inline apps/condo/domains/user/components/UnitNameInput.tsx:14]: absolute import plz

File: apps/condo/domains/ticket/components/TicketId/TicketClassifierField.tsx [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+import { Breadcrumb, Typography } from 'antd'
+import { BaseType } from 'antd/lib/typography/Base'
+import { compact, get } from 'lodash'
+
+import { useIntl } from '@core/next/intl'
+import { Ticket } from '@app/condo/schema'
+
+import { PageFieldRow } from '@condo/domains/common/components/PageFieldRow'
+
+const CLASSIFIER_VALUE_STYLE = { fontSize: '16px' }
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketId/TicketClassifierField.tsx:11]: plz use font sizes from global styles

File: apps/condo/domains/ticket/components/TicketId/TicketPropertyField.tsx [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+import { useIntl } from '@core/next/intl'
+import { Ticket } from '@app/condo/schema'
+
+import { PageFieldRow } from '@condo/domains/common/components/PageFieldRow'
+import { getAddressDetails } from '@condo/domains/common/utils/helpers'
+import { TicketPropertyHintCard } from '@condo/domains/ticket/components/TicketPropertyHint/TicketPropertyHintCard'
+import { TICKET_CARD_LINK_STYLE } from '@condo/domains/ticket/constants/style'
+import { PARKING_SECTION_TYPE } from '../../../property/constants/common'
+
+const HINT_CARD_STYLE = { maxHeight: '5em ' }
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketId/TicketPropertyField.tsx:16]:
```suggestion
const HINT_CARD_STYLE = { maxHeight: '5em' }
```

File: apps/condo/domains/ticket/components/TicketId/TicketPropertyField.tsx [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+import { Gutter } from 'antd/es/grid/row'
+
+import { useIntl } from '@core/next/intl'
+import { Ticket } from '@app/condo/schema'
+
+import { PageFieldRow } from '@condo/domains/common/components/PageFieldRow'
+import { getAddressDetails } from '@condo/domains/common/utils/helpers'
+import { TicketPropertyHintCard } from '@condo/domains/ticket/components/TicketPropertyHint/TicketPropertyHintCard'
+import { TICKET_CARD_LINK_STYLE } from '@condo/domains/ticket/constants/style'
+import { PARKING_SECTION_TYPE } from '../../../property/constants/common'
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketId/TicketPropertyField.tsx:14]: absolute import

File: apps/condo/domains/ticket/components/TicketId/TicketUserInfoField.tsx [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+interface ITicketUserInfoFieldProps {
+    user?: Partial<User>
+}
+
+export const TicketUserInfoField: React.FC<ITicketUserInfoFieldProps> = (props) => {
+    const id = useMemo(() => get(props, ['user', 'id']), [props])
+    const name = useMemo(() => get(props, ['user', 'name']), [props])
+    const phone = useMemo(() => get(props, ['user', 'phone']), [props])
+    const email = useMemo(() => get(props, ['user', 'email']), [props])
+    const userInfo = useMemo(() => [], [])
```
Comment (sitozzz) [inline apps/condo/domains/ticket/components/TicketId/TicketUserInfoField.tsx:19]: what meaning of this line?

File: apps/condo/domains/ticket/constants/style.js [hunk 10]
```diff
@@ -1,4 +1,5 @@
 const { CANCELED, COMPLETED, DEFERRED, NEW_OR_REOPEND, PROCESSING, CLOSED } = require('./statusTypes')
+const { colors } = require('../../common/constants/style')
```
Comment (sitozzz) [inline apps/condo/domains/ticket/constants/style.js:2]: absolute import

**Pull request merged**